### PR TITLE
Production method additions

### PR DESCRIPTION
**Pull request by JackEPepper**

### CODE REVIEW

File: localization/english/CWP/CWP_production_methods_l_english.yml [hunk 1]
```diff
@@ -902,14 +910,63 @@
   pmg_ownership_telecommunications_industry: "Ownership"
   pmg_base_tourism: "Tourism"
   pmg_ownership_tourism: "Ownership"
-  pmg_base_financial_services: "Financial Services"
+  pmg_base_financial_services: "Personal Services"
```
Comment (Arcadyi) [inline localization/english/CWP/CWP_production_methods_l_english.yml:913]: Why was this changed?

**Review by Arcadyi — APPROVED** [review]

File: localization/english/CWP/CWP_production_methods_l_english.yml (hunk 1 above)
Comment (JackEPepper) [reply localization/english/CWP/CWP_production_methods_l_english.yml:913]: To differentiate personal financial services and the new commercial services PMG.

File: common/production_methods/06_urban_center.txt [hunk 2]
```diff
@@ -760,9 +752,6 @@ pm_enhanced_geothermal_systems = {
 pm_nuclear_plant = {
 	texture = "gfx/interface/icons/production_method_icons/power_plant.dds"
 	pollution_generation = 10
-	unlocking_technologies = {
-		nuclear_fission
```
Comment (DZappala) [inline common/production_methods/06_urban_center.txt:764]: surely this shouldn't be allowed at game start.

**Review by DZappala — APPROVED** [review]
couple of changes needed. but otherwise all goods.

File: common/production_methods/06_urban_center.txt (hunk 2 above)
Comment (JackEPepper) [reply common/production_methods/06_urban_center.txt:764]: Since its the first PM for the PMG, it shouldn't have a technology requirement.  The tech requirement is on the building itself, not the PM.

Comment (DZappala) [reply common/production_methods/06_urban_center.txt:764]: copy that 🫡

File: common/production_methods/03_mines.txt [hunk 3]
```diff
@@ -1378,8 +1378,7 @@ pm_traditional_tools = {
 #### Ownership
 pm_privately_owned_mine = {
 	texture = "gfx/interface/icons/production_method_icons/privately_owned.dds"
-	ai_weight = 10000
-	unlocking_production_methods = {
+		unlocking_production_methods = {
```
[the remark this answers is not in the document]
Comment (JackEPepper) [reply common/production_methods/03_mines.txt]: fixed
